e2e/async-regenerator/package.json

```diff
@@ -6,6 +6,9 @@
     "@babel/polyfill": "^7.2.5"
   },
   "jest": {
-    "testEnvironment": "node"
+    "testEnvironment": "node",
+    "transformIgnorePatterns": [
+      "jest-jasmine2"
+    ]
```

Comment:
@SimenB This was the reason the last test was failing. Adding it manually to `transformIgnorePatterns`, again as mentioned in a comment below these ignores are needed only when testing the jest repo, any consumers will not need to add these as these will be within node_modules